gtsam/geometry/Pose3.h

```diff
@@ -44,7 +44,7 @@ class GTSAM_EXPORT Pose3: public LieGroup<Pose3, 6> {
 private:
 
   Rot3 R_; ///< Rotation gRp, between global and pose frame
-  Point3 t_; ///< Translation gTp, from global origin to pose frame origin
+  Point3 t_; ///< Translation gPp, from global origin to pose frame origin
```

Comment:
I am not sure about this

Reply:
I had asked about this and @Alescontrela gave me a paper reference where they use this notation.